src/styles/base.js

```diff
@@ -1494,6 +1498,7 @@ export const themeLT = {
   sliderBarColor: '#A0A7B4',
   sliderBarBgd: '#D3D8E0',
   sliderHandleColor: '#F7F7F7',
```

Comment:
You created 4 new theme names
```
dataTabletextColor
sliderInactiveBorderColor
panelTabColor
dataTabletextColor
```

Stll missing some in `temeLT` and `themeBS`
Be diligent please

Reply:
Both themeLT and ThemeBS extends from `theme`. the variables has been added there(in theme). should I add them again in both themes?

Reply:
for the ones share same value between dark and light, you don't need to, but for the ones have different values you need to add them to light specifically

Reply:
This change was attended